src/app/lib/config/services/ws.ts

```diff
@@ -395,11 +413,15 @@ export const service: DefaultServiceConfig = {
             id: 'kyrgyz',
             href: 'https://www.bbc.com/kyrgyz',
             label: 'Кыргыз тилиндеги жаңылыктар',
+            lang: 'ky',
+            latinTransliteration: 'Kyrgyz tilindegi zhaniliktar',
           },
           {
             id: 'uzbek',
             href: 'https://www.bbc.com/uzbek',
             label: 'Ўзбек тилидаги янгиликлар',
+            lang: 'uz',
```

Comment:
Does this need to match the lang of the default variant?
```suggestion
            lang: 'uz-latn',
```